main-app/app/scripts/filters/filters.js

```diff
@@ -9,6 +9,9 @@
                 else if (input.playerNum == input.noughtPlayer){
                     return 'image nought';
                 }
+                else {
+                    return null;
```

Comment:
I'm ambivalent about this - seems needless or wrong. Either:
1. Ignore this condition
2. Return ''
3. Return the original value
4. Return the string 'image'  

Whatever you decide on, do it and feel free to merge it yourself